prefq/examples/imitation_preference_comparisons.py

```diff
@@ -61,8 +235,13 @@
     preference_querent=querent,
     preference_gatherer=gatherer,
     reward_trainer=reward_trainer,
+    fragment_length=100,
+    transition_oversampling=1,
+    initial_comparison_frac=0.1,
+    allow_variable_horizon=False,
```

Comment:
Why these changes?

Reply:
these changes have been copied from one of the examples provided by the authors of the PrefCollectGatherer: 

https://github.com/HumanCompatibleAI/imitation/blob/4dbf20c6d42286377901dd538cd6c5001225fa40/docs/tutorials/5b_train_preference_comparisons_with_synchronous_human_feedback.ipynb